mission/task_manager/scripts/test_2_node.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
```

Comment:
remove test node

Reply:
made a folder "example_nodes" for the test nodes so that the nodes can be used as examples when setting up the real task nodes